test/unit/server/server_spec.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+var Rest        = require("../../../lib/plugins/rest");
+var Sinon       = require("sinon");
+var expect      = require("chai").expect;
+
+describe("The server", function () {
+	var restStub;
+	var server;
+	var DB_URL;
+	before(function () {
+		DB_URL = process.env.DB_URL;
+		process.env.DB_URL = undefined;
```

Comment:
Manipulating the `DB_URL` should no longer be necessary since we've eliminated that branch.